### DISCUSSION TITLE
NocoDB App Builder

### DISCUSSION THREAD

**Discussion by Vadorequest** [nocodb/nocodb#656] [Features]
I'd like to know if there exist any "App builder" on top of NocoDB.

As it exists several App builders on top of Airtable, such as:
- Softr
- Pory
- StackerHQ (I use that one)

Those tools help transform DBs like Airtable and NocoDB into actual apps, and it's a big requirement for us to find a DB that can be easily turned into an actual app.

My current tools are Airtable and Stacker, but as I'm trying to move away from Airtable (NocoDB <3) I'll need an alternative to Stacker, too.

I haven't searched if some tool already make that possible on top of NocoDB. Do you know any?

**Comment by include** [comment]
@Vadorequest maybe appsmith ?

**Comment by jingleberry11** [comment]
I also like to know if there exist any "App builders" that uses NocoDB as source? softr is dank, but only support airtable.
